patterns/1-initial/sustainable-innersource-program.md

```diff
@@ -0,0 +1,73 @@
+## Title
+
+Sustainable InnerSource Program
+
+## Patlet
+
+I've started an InnerSource program - what next? How do I ensure InnerSource remains a part of how we create software in the long term?
+
+## Problem
+
+Given an InnerSource program and/or community has been started within your organisation, you are unsure how to scale and/or sustain to ensure that InnerSource remains part of culture and best practice in the long term. When scaling and sustaining an InnerSource program fails, it leads to problems including unusable projects/products, an unengaged community, and loss of support from leadership.
+
+## Context
+
+* The problem exists in fledgling InnerSource programs/communities
+* You may not have full time support from an InnerSource or Open Source Program Office (ISPO/OSPO)
+* There is no clear vision on the future of the program
+* You are worried that key people will move on to new roles, momentum will be lost and silently die, or future leaders will stop supporting InnerSource.
+* You are an individual or organization who is concerned with the fragile state and future of highly-used and impactful InnerSource projects.
+[1] Sustain. 2017. [https://sustainoss.org/assets/pdf/SustainOSS-west-2017-report.pdf](https://sustainoss.org/assets/pdf/SustainOSS-west-2017-report.pdf)
+* There is increasing quantity and decreasing quality of InnerSource projects at your organization are changing.
```

Comment:
```suggestion
* There is increasing quantity and decreasing quality of InnerSource projects at your organization.
```

Looks like the "are changing" was a remainder of a previous rewrite of this sentence?